docs/topics/proc_deploying-an-existing-application-to-openshift.adoc

```diff
@@ -33,5 +33,12 @@ endif::[]
 
 +
 In this profile, the Fabric8 Maven plugin is invoked for building and deploying the application to OpenShift.
-
+. Create a [filename]`deployment.yaml` file in the [filename]`src/main/fabric8` directory and add the JAVA_OPTIONS shown in the following example.
```

Comment:
Is there a particular reason you've opted to use semantic markup?
It is absolutely OK to do so, I'm just wondering.
(while this is 100% AsciiDoctor-compatible, I'm unaware of any tools  that would require it at the moment).

Reply:
Mostly out of curiosity as I was looking to see if there would be a rendered difference for the the same markup. I tend to favor semantic distinctions as they can be important for L10N. I didn't see a distinction in my local build.

Reply:
I've been thinking about starting using semantic markup, and I'm very much open to the idea. RHEL docs do that, but many other products don't. Having said that, because we don't use semantic markup anywhere else in the doc set yet, I'd rather you didn't use it here.